### PR TITLE
Add some scripts that let us build reproducible tarballs.

### PR DESCRIPTION
**Pull request by dlorenc**
Ref #74

### CODE REVIEW

**Comment by dlorenc** [comment]
This still needs a bit of work before we can switch to using it, but it's in a good spot now.

File: reproducible/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ The same git revision should result in a Docker image with the same digest, ever
 
 ### Usage
 
-Use `bazel build //repeatable:debian8` to generate the image locally, or `gcloud container builds submit --config=repeatable/cloudbuild.yaml .` to build it in the cloud.
+Use `gcloud container builds submit --config=repeatable/cloudbuild.yaml .` to build the image in the cloud.
```
Comment (sharifelgamal) [inline reproducible/README.md]: You reference a "repeatable" directory in a few spots. Should those be "reproducible" or is there a separate directory?

Comment (dlorenc) [inline reproducible/README.md]: Fixed.

**Comment by dlorenc** [comment]
PTAL

**Comment by dlorenc** [comment]
Switched this over to use a custom rule. Should be much easier to understand now.

**Pull request merged**